code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -375,6 +375,7 @@
 	preloaded_reagents = list("doctorsdelight" = 8, "tricordrazine" = 8)
 	bitesize = 3
 	cooked = TRUE
+	taste_tag = list(VEGETARIAN_FOOD,VEGAN_FOOD)
```

Comment:
You cannot put food labels on reagents. If you are going to do it you will have to work a lot more than this. At the moment only the ethanol subtypes.

Reply:
oh wait. only ethanol.. why? 
can you point me the root of it. so i can get it to work with reagents

Reply:
but WAIT
at the line "23"
the /obj/item/weapon/reagent_containers/food/snacks
have a var for taste tag

and that was put there by yourself

Reply:
but WAIT
at the line "23"
the /obj/item/weapon/reagent_containers/food/snacks
have a var for taste "var/list/taste_tag = list(BLAND_FOOD)"

and that was put there by yourself.  (i'm a shit coder so i'm very confused)

Reply:
forgive me, I must rest and I am seeing things that are not.

Reply:
its ok. your point was helpfull nonetheless and also the points made in discord. i thank you a lot for it